src/drivers/auth0Driver.ts

```diff
@@ -58,6 +64,7 @@ export const makeAuth0Driver = (
       login$: Stream.create(),
       logout$: Stream.create(),
       requiredLogin$: Stream.createWithMemory(),
+      appState$: Stream.createWithMemory()
```

Comment:
```suggestion
      appState$: Stream.create()
```

`createWithMemory` だとページが切り替わるタイミングで都度流れてしまう気がしました。